test/unit/itext.js

```diff
@@ -105,7 +105,10 @@
       assert.ok(typeof fabric.IText.fromObject === 'function');
       fabric.IText.fromObject(ITEXT_OBJECT).then(function(iText) {
         assert.ok(iText instanceof fabric.IText);
-        assert.deepEqual(ITEXT_OBJECT, iText.toObject());
+        // change styles from array to object for comparison
+        var object = iText.toObject();
+            object.styles = {};
+        assert.deepEqual(ITEXT_OBJECT, object);
```

Comment:
Very minor and insignificant... but if you want
```suggestion
        assert.deepEqual(object, ITEXT_OBJECT);
```

Reply:
👍